src/Parser/BalanceDruid/Modules/Features/AlwaysBeCasting.js

```diff
@@ -35,7 +35,7 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
       .addSuggestion((suggest, actual, recommended) => {
         return suggest(<span> Your downtime can be improved. Try to Always Be Casting (ABC)...</span>)
           .icon('spell_mage_altertime')
-          .actual(`${formatPercentage(this.deadTimePercentage)}% dead GCD time`)
+          .actual(`${formatPercentage(actual)}% downtime`)
           .recommended(`${Math.round(formatPercentage(recommended))}% is recommended`)
```

Comment:
Missing `<` (so it becomes `<2% is recommended` instead of `2% is recommended` (technically it's `<=2% is recommended` but let's not overcomplicate it :P)